dmscripts/generate_supplier_user_csv.py

```diff
@@ -10,7 +10,6 @@ def generate_supplier_csv(framework_slug, data_api_client, logger):
     supplier_rows = data_api_client.export_suppliers(framework_slug).get('suppliers', [])
     if not supplier_rows:
         logger.warn('No supplier data found for framework {}'.format(framework_slug))
-        return [], [], None
```

Comment:
Can we instead add a line above the `if not download_filename:` clause on L146. Something like

```
if not rows:
    logger.info("No data for {}".format(whatever_it_is))
    return True
```

We also have tests for this script - should probably update them with the new expected behaviour.

Reply:
Is the idea to not upload empty files? I think in that case it would make more sense to put the test in `generate_csv_and_upload_to_s3`?

Reply:
Added a commit with a short-circuit in `generate_csv_and_upload_to_s3` https://github.com/alphagov/digitalmarketplace-scripts/pull/361/commits/e33c190aeeb74545bba669b44aeac32ab178fe2e